### DISCUSSION TITLE
/turing/conversation/create: failed to parse response body.

### DISCUSSION THREAD

**Discussion by beiwushijie** [waylaidwanderer/node-chatgpt-api#379] [Help] [answered]
I now get this error no matter what question I ask, I tried restarting and clearing the dialog but it still doesn't work, how can this be fixed
<img width="1440" alt="image" src="https://github.com/waylaidwanderer/node-chatgpt-api/assets/61958116/ba320e4a-c1dc-42e2-9d70-6fcff7858cd6">

**Comment by muhaoran1987** [comment, marked as answer]
同样的问题，不知是微软开始封禁了还是海底光缆挖断了，fb和Google今天也是各种崩

**Reply by beiwushijie** [reply]
it can solve the problem

**Reply by muhaoran1987** [reply]
你那边恢复了吗？

**Reply by beiwushijie** [reply]
恢复了，直接换成cookie就能用了

**Reply by muhaoran1987** [reply]
老哥方便加个微信不：muhaoran0000

**Reply by beiwushijie** [reply]
晚安小圆 ok
